rat/variable_table.py

```diff
@@ -1,344 +1,295 @@
-from dataclasses import dataclass, field
-from enum import Enum
-import numpy
+from collections import namedtuple
+from typing import Any, Dict, List, Tuple, Iterable, Union, TypeVar, NamedTuple
+
 import pandas
-import pprint
-from typing import Dict, Set, Tuple, Iterable, Union
+from sortedcontainers import SortedDict, SortedSet
+
+K = TypeVar("K")
+V = TypeVar("V")
 
-from .exceptions import CompileError, MergeError
 
+def get_dataframe_name_by_variable_and_column_name(
```

Comment:
I've updated this file so it supports multiple input files that share the same column names. In addition I've separated passing columns as variable names/subscript names so it tells you on error whether it couldn't find a subscript or data variavle column.